lib/bundles/inspec-compliance/cli.rb

```diff
@@ -198,6 +228,29 @@ def logout
 
     private
 
+    def login_automate_config(url, user, dctoken, usertoken, ent)
+      config = Compliance::Configuration.new
+      config['server'] = url
+      config['ent'] = ent
```

Comment:
We should store all automate specific config under the section `config['automate']`